idea/idea-completion/src/org/jetbrains/kotlin/idea/completion/KeywordCompletion.kt

```diff
@@ -103,7 +103,8 @@ object KeywordCompletion {
             var keyword = keywordToken.value
 
             val nextKeyword = when {
-                keywordToken == SUSPEND_KEYWORD && ((position.containingFile as? KtFile)?.isScript() == true) -> null
+                keywordToken == SUSPEND_KEYWORD && ((position.containingFile as? KtFile)?.isScript() == true 
```

Comment:
It seems like there are no actual need to prevent `suspend` or `suspend fun` completion in `.kts` files. Could you please remove this check all along, so it will be something like: 

```kt
val nextKeyword = when {
    keywordToken == SUSPEND_KEYWORD && position.getStrictParentOfType<KtTypeReference>() != null -> null
    else -> COMPOUND_KEYWORDS[keywordToken]
}
```